src/App.jsx

```diff
@@ -1,34 +1,32 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import './App.css'
+import { useState } from "react";
+import TodoInput from "./TodoInput";
+import TodoList from "./TodoList";
+import "./App.css";
 
-function App() {
-  const [count, setCount] = useState(0)
+const App = () => {
+  const [todos, setTodos] = useState([]);
+  const [todo, setTodo] = useState("");
+
+  const addTodo = () => {
+    if (todo !== "") {
+      setTodos([...todos, todo]);
+      setTodo("");
+    }
+  };
+  const deleteTodo = (text) => {
+    const newTodos = todos.filter((todo) => {
+      return todo !== text;
+    });
+    setTodos(newTodos);
```

Comment:
This is fine if the goal is to delete all tasks with the same name. i.e. Multiple tasks named "a" will all be deleted if a single one is chosen to be deleted.

I suggest using id's assigned to each task so that they are all distinct and deleting them by id.